### PR TITLE
Encointer update

### LINKED ISSUE #3719: Encointer is now live
https://wiki.polkadot.network/docs/learn-common-goods#encointer

Encointer is now live. Please update to remove the Note and change the tense to present.

### PR DESCRIPTION
**Pull request by emresurmeli**
Addresses #3719

cc: @brenzi

### CODE REVIEW

File: docs/learn/learn-common-good-chains.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Statemint will require the Relay Chain's "root origin", i.e. a referendum. Some
 
 ### Encointer
 
-Encointer will become Kusama's second common good parachain, which adds logic to the Relay Chain
+Encointer is a blockchain platform for self-sovereign ID and a global universal basic income. With [referendum 187](https://kusama.polkassembly.io/referendum/187) Encointer became Kusama's second common good parachain and it adds logic to the Relay Chain
```
Comment (joepetrowski) [inline docs/learn/learn-common-good-chains.md]: @brenzi ?

Comment (profradha) [inline docs/learn/learn-common-good-chains.md]: Encointer was registered as a common good parachain through this referendum https://kusama.polkassembly.io/referendum/158 and got upgraded to its full functionality through the linked referendum. Rephrase it accordingly if needed to reflect both referenda.

```suggestion
Encointer is a blockchain platform for self-sovereign ID and a global universal basic income. With [referendum 158](https://kusama.polkassembly.io/referendum/158) Encointer became Kusama's second common good parachain and it adds logic to the Relay Chain
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Minor edit

File: docs/learn/learn-common-good-chains.md (hunk 1 above)
Comment (emresurmeli) [reply docs/learn/learn-common-good-chains.md]: Yeah I think both will be helpful

**Review by DrW3RK — APPROVED** [review]
Looks good. Please run prettify and then merge

**Pull request merged**